package/src/constructs/alert-channel.ts

```diff
@@ -1,13 +1,36 @@
 import { Construct } from './construct'
 
 export interface AlertChannelProps {
-    sendRecovery: boolean
-    sendFailure: boolean
-    sendDegraded: boolean
-    sslExpiry: boolean
-    sslExpiryThreshold: number
-  }
+  /**
+   * Determines if an alert should be send for check recoveries.
+   */
+  sendRecovery: boolean
+  /**
+   * Determines if an alert should be send for check failures.
+   */
+  sendFailure: boolean
+  /**
+   * Determines if an alert should be send when a check is degraded.
+   */
+  sendDegraded: boolean
+  /**
+   * Determines if an alert should be send for expiring SSL certificates.
+   */
+  sslExpiry: boolean
+  /**
+   * At what moment in time to start alerting on SSL certificates.
+   */
+  sslExpiryThreshold: number
+}
 
+/**
+ * Creates an Alert Channels
+ *
+ * @remarks
+ *
+ * This class make use of the Alert Channels endpoints
+ * listed {@link https://developers.checklyhq.com/reference/postv1alertchannels here}
```

Comment:
This is not true. We have different schemas for these constructs. I would not put the public api docs here. We can add the link when we put project/deploy behind v1 as well

Reply:
Sounds good 👍